src/act.other.cpp

```diff
@@ -4456,12 +4456,13 @@ ACMD(do_spray)
         return;
       }
       struct obj_data *paint = read_object(OBJ_GRAFFITI, VIRTUAL);
-      snprintf(buf, sizeof(buf), "a piece of graffiti that says \"%s^n\"", argument);
+      snprintf(buf, sizeof(buf), "^g   %s^n", argument);
```

Comment:
This is the item name itself, it should be left as the piece of graffiti string since it doesn't show up in the room.